dart_test.yaml

```diff
@@ -0,0 +1,27 @@
+# Dart 1:
+# -------
+#
+# Run unit tests:
+#
+#     $ pub run test
+#
+# Dart 2:
+# -------
+#
+# Run unit tests:
+#
+#     $ pub run build_runner test
+platforms:
+- chrome
+- vm
+
+paths:
+- test/unit/browser
+- test/unit/vm
+
+presets:
+  dart2:
+    exclude_tags: "no-dart2"
```

Comment:
There aren't any tests that are tagged with `no-dart2` so you could remove this preset if you want.